pandas/core/generic.py

```diff
@@ -1109,10 +1141,19 @@ def to_json(self, path_or_buf=None, orient=None, date_format='epoch',
               - index : dict like {index -> {column -> value}}
               - columns : dict like {column -> {index -> value}}
               - values : just the values array
+              - table : dict like {'schema': {schema}, 'data': {data}}
+              ``{'schema': [schema], 'data': [data]}``
```

Comment:
This line needs 2 extra spaces I think to align 
(and it seems a bit repetition of the line above?)